lib/main.dart

```diff
@@ -69,7 +69,7 @@ class _MyHomePageState extends State<MyHomePage> {
       alpha2 = 'Failed to get sim country code.';
     }
 
-    switch (alpha2) {
+    switch (alpha2?.toLowerCase()) {
```

Comment:
Android and iOS _should_ return the same values, but they don't . iOS => 'us' and Android => 'US'. Don't know until you know, you know?